mesonbuild/mtest.py

```diff
@@ -1120,9 +1121,10 @@ async def read_decode(reader: asyncio.StreamReader, console_mode: ConsoleUser) -
             except asyncio.LimitOverrunError as e:
                 line_bytes = await reader.readexactly(e.consumed)
             line = decode(line_bytes)
-            stdo_lines.append(line)
-            if console_mode is ConsoleUser.STDOUT:
-                print(line, end='', flush=True)
+            if line:
```

Comment:
Why the "if line:" addition? Please explain in the commit message (but it seems wrong).

Reply:
Explained.